bucket/camunda-operate.json

```diff
@@ -0,0 +1,27 @@
+{
+    "homepage": "https://github.com/zeebe-io/zeebe",
+    "version": "1.2.0",
+    "license": "Zeebe Community License Version 1.0",
+    "description": "Monitor, manage and troubleshoot running workflow instances.",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/zeebe-io/zeebe/releases/download/0.22.1/camunda-operate-1.2.0.zip",
+            "hash": "b236f41f1085199e0edfcd12f013b6b2dc8844eeee5a3e54ffb53047e700ebb3"
+        }
+    },
+    "extract_dir": "camunda-operate-1.2.0",
+    "bin": "bin\\operate.bat",
+    "checkver": {
+        "url": "https://github.com/zeebe-io/zeebe/releases",
+        "regex": "https://github.com/zeebe-io/zeebe/releases/download/(?<release>[\\d.]+)/camunda-operate-(?<version>[\\d.]+).zip",
+        "replace": "$matchVersion",
+        "extract_dir": "camunda-operate-$matchVersion"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/zeebe-io/zeebe/releases/download/$matchRelease/camunda-operate-$matchVersion.zip"
+            }
+        }
+    }
+}
```

Comment:
```suggestion
{
    "version": "1.2.0",
    "description": " Workflow Engine for Microservices Orchestration",
    "homepage": "https://zeebe.io",
    "license": {
        "identifier": "Zeebe Community License Version 1.0",
        "url": "https://zeebe.io/legal/zeebe-community-license-1.0/"
    },
    "architecture": {
        "64bit": {
            "url": "https://github.com/zeebe-io/zeebe/releases/download/0.22.1/camunda-operate-1.2.0.zip",
            "hash": "sha1:b8e6330d5e2c917250add386b8077524fa74c54e"
        }
    },
    "extract_dir": "camunda-operate-1.2.0",
    "bin": [
        [
            "bin\\operate.bat",
            "comunda-operate"
        ]
    ],
    "checkver": {
        "github": "https://github.com/zeebe-io/zeebe",
        "regex": "download/(?<release>[\\d.]+)/camunda-operate-([\\d.]+)\\.zip"
    },
    "autoupdate": {
        "architecture": {
            "64bit": {
                "url": "https://github.com/zeebe-io/zeebe/releases/download/$matchRelease/camunda-operate-$version.zip"
            }
        },
        "hash": {
            "url": "$url.sha1sum"
        },
        "extract_dir": "camunda-operate-$version"
    }
}
```

Reply:
It says "Workflow Engine for Microservices Orchestration" on the GitHub page, but it's actually the tool to "Monitor, manage and troubleshoot running workflow instances". I don't know why they decided to distribute both tools within one repository.

![Camunda Cloud](https://camunda.com/img/products/cloud/camunda-cloud-stack-web.svg)

Reply:
Then apply these changes and find the correct homepage.

Reply:
Camunda Cloud is an SaaS version of Zeebe + Operate (+ Cawemo + Optimize).

Operate is called Camunda Operate right now, even though it is in the Zeebe (which is a Camunda product as well) repository. It's similar to the Camunda Cockpit (https://camunda.com/products/cockpit/), but it's not the same.

IMHO it's either https://zeebe.io / https://docs.zeebe.io/operate-user-guide/index.html or nothing. There is no dedicated page for Operate.